orderly_web/deploy.py

```diff
@@ -0,0 +1,103 @@
+import docker
+
+from orderly_web.docker_helpers import docker_client, \
+    ensure_network, ensure_volume, \
+    exec_safely, string_into_container
+
+
+def deploy(cfg):
+    with docker_client() as cl:
+        ensure_network(cl, cfg.network)
```

Comment:
So the network and volume creation are idempotent, which makes sense since we might be hooking into an existing network, but then the container creation is not and if containers are already running you'll get an error. Which also maybe makes sense since the containers are the bits getting re-deployed. But perhaps we should either stop them automatically or return a friendly "you can't do this, please run `orderly-web stop` before re-deploying" message?

Reply:
Minimal work on this here: 4bd5b1b - I agree that something that will "do the right thing" with respect to upgrading etc would be nice to have in place